From b8d9a1cc904d9793877b7af2940d0017a7bc0def Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Mon, 8 Apr 2024 15:43:31 +0200
Subject: [PATCH 0811/1695] core: drivers: update stm32mp_dt_bindings.h with
 TAMP bindings

Add the inclusion of TAMP bindings to this file.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 1bf87989515c896a6baaca489f3714b31043a9f8 core: drivers: update stm32mp_dt_bindings.h with TAMP bindings]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: I5acd9f6505d9682dafb6b50317743e60b72f11d9
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/372210
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
Tested-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
---
 core/include/drivers/stm32mp_dt_bindings.h | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/core/include/drivers/stm32mp_dt_bindings.h b/core/include/drivers/stm32mp_dt_bindings.h
index d0f16b570..822af238d 100644
--- a/core/include/drivers/stm32mp_dt_bindings.h
+++ b/core/include/drivers/stm32mp_dt_bindings.h
@@ -13,6 +13,7 @@
 #include <dt-bindings/reset/stm32mp13-resets.h>
 #include <dt-bindings/soc/stm32mp13-etzpc.h>
 #include <dt-bindings/soc/stm32mp13-tzc400.h>
+#include <dt-bindings/tamper/st,stm32mp13-tamp.h>
 #endif
 
 #ifdef CFG_STM32MP15
@@ -23,6 +24,7 @@
 #include <dt-bindings/reset/stm32mp1-resets.h>
 #include <dt-bindings/soc/stm32mp15-etzpc.h>
 #include <dt-bindings/soc/stm32mp15-tzc400.h>
+#include <dt-bindings/tamper/st,stm32-tamp.h>
 #endif
 
 #if defined(CFG_STM32MP25) || defined(CFG_STM32MP23)
@@ -33,6 +35,7 @@
 #include <dt-bindings/soc/stm32mp25-rifsc.h>
 #include <dt-bindings/soc/stm32mp25-risab.h>
 #include <dt-bindings/soc/stm32mp25-risaf.h>
+#include <dt-bindings/tamper/st,stm32mp25-tamp.h>
 #endif
 
 #ifdef CFG_STM32MP21
@@ -42,6 +45,7 @@
 #include <dt-bindings/soc/stm32mp25-rif.h>
 #include <dt-bindings/soc/stm32mp21-rifsc.h>
 #include <dt-bindings/soc/stm32mp25-risaf.h>
+#include <dt-bindings/tamper/st,stm32mp21-tamp.h>
 #endif
 
 #endif /* __DRIVERS_STM32MP_DT_BINDINGS_H */
-- 
2.47.2

